obspy/imaging/tests/test_scan.py

```diff
@@ -109,8 +91,8 @@ def test_multipleSamplingrates(self):
                         fp.flush()
                         fp.seek(0)
                         files.append(fp.name)
-                    with ImageComparison(self.path, 'scan_mult_sampl.png',
-                                         reltol=reltol) as ic:
+                    with ImageComparison(self.path, 'scan_mult_sampl.png') \
```

Comment:
Might as well correct the file name to `scan_multi_sampling` or something, since it's too long for here anyway.